MediaBrowser.Model/Drawing/ImageResolution.cs

```diff
@@ -0,0 +1,53 @@
+namespace MediaBrowser.Model.Drawing
+{
+    /// <summary>
+    /// Enum ImageResolution.
+    /// </summary>
+    public enum ImageResolution
+    {
+        /// <summary>
+        /// MatchSource.
+        /// </summary>
+        MatchSource,
```

Comment:
Enum values should have associated values
```suggestion
        MatchSource = 0,
```

Reply:
I've added the values.